src/current/v22.1/operational-faqs.md

```diff
@@ -39,7 +39,20 @@ Given the 10 day accumulation period, you can expect storage per node to increas
 
 or about 6 GiB. With on-disk compression, the actual disk usage is likely to be about 4 GiB.
 
-However, depending on your usage of time-series charts in the [DB Console](ui-overview-dashboard.html), you may prefer to reduce the amount of disk used by time-series data. To reduce the amount of time-series data stored, or to disable it altogether, see [Can I reduce or disable the storage of time-series data?](#can-i-reduce-or-disable-the-storage-of-time-series-data)
+However, depending on your usage of time-series charts in the [DB Console](ui-overview-dashboard.html), you may prefer to reduce the amount of disk used by time-series data. To reduce the amount of time-series data stored, or to disable it altogether, see [Can I reduce or disable the storage of time-series data?](#can-i-reduce-or-disable-the-storage-of-time-series-data).
```

Comment:
Since the link at the end of paragraph ends with a question mark, it is doubly punctuated? Is that OK? My feeling is that it looks strange, but could also see an argument for keeping. Also, Matt has made me realize that changing "see" to "refer to" is a positive swap.

```suggestion
However, depending on your usage of time-series charts in the [DB Console](ui-overview-dashboard.html), you may prefer to reduce the amount of disk used by time-series data. To reduce the amount of time-series data stored, or to disable it altogether, refer to [Can I reduce or disable the storage of time-series data?](#can-i-reduce-or-disable-the-storage-of-time-series-data)
```

Reply:
Good points, updated!